content/pl/backing-services.md

```diff
@@ -0,0 +1,15 @@
+## IV. Usługi wspierające
+### Traktuj usługi wspierające jako przydzielone zasoby
+
+Usługą wspierająca to każda, z której aplikacja korzysta przez sieć jako część normalnego działania. Przykłady zawierają magazyny danych (takie jak [MySQL](http://dev.mysql.com/) albo [CouchDB](http://couchdb.apache.org/)), systemy wysyłanie/kolejkowania wiadomości (takie jak [RabbitMQ](http://www.rabbitmq.com/) czy [Beanstalkd](http://kr.github.com/beanstalkd/)), usługi SMTP do zewnętrznej wysyłki emaili (np. [Postfix](http://www.postfix.org/)), oraz systemy cachowania pamięci (np. [Memcached](http://memcached.org/)).
```

Comment:
1. `Usługą` -> no to albo `wspierającą` albo wtedy `Usługa` . 
2. `s/Przykłady zawierają/Zaliczamy do nich np. magazyny danych`
3. `systemy wysyłanie/kolejkowania wiadomości` -> `wysyłania`